tensorflow_addons/layers/tests/spatial_pyramid_pooling_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for AdaptivePooling layers."""
+
+import pytest
+from tensorflow_addons.layers.spatial_pyramid_pooling import SpatialPyramidPooling2D
+
```

Comment:
Please consider adding more tests around config, serializability, dtype and functionality checks.

Reply:
I've added the tests.